src/nurse/views.py

```diff
@@ -18,6 +22,77 @@
 )
 
 
+class SendEmailToDoctorView(APIView):
+    def post(self, request, pk):
+        prescription = get_object_or_404(Prescription, id=pk)
+
+        patient = prescription.patient
+        if not patient:
+            return Response(
+                {"error": "This prescription has no patient"},
+                status=status.HTTP_404_NOT_FOUND,
+            )
+
+        doctor_name = prescription.prescribing_doctor
+        email_doctor = prescription.email_doctor
+        if not email_doctor:
+            return Response(
+                {"error": "Doctor has no email"}, status=status.HTTP_400_BAD_REQUEST
+            )
+
+        user = request.user
+        try:
+            nurse = Nurse.objects.get(user=user)
+        except Nurse.DoesNotExist:
+            return Response(
+                {"error": "You are not authorized to send emails for this patient"},
+                status=status.HTTP_403_FORBIDDEN,
+            )
```

Comment:
No user and associated with a nurse I don't want to create if she has no nurse because it means that a user who is not a nurse tries to send email

Reply:
Aren't all users nurses in the application? And we already check if we have access to the patient aren't we?